libnet/Makefile.am.common

```diff
@@ -5,7 +5,6 @@
 # All rights reserved.
```

Comment:
Would also be my inclination. Its very minimal functionality can be migrated away. Except for the *~ business, which is an emacs-ism and hence belongs in per-user configuration files. (I'm an emacs fanatic and have such in my own per-user global git ignore etc.)